bats/core/api/user.bats

```diff
@@ -24,3 +24,37 @@ setup_file() {
   language="$(graphql_output '.data.me.language')"
   [[ "$language" == "$new_language" ]] || exit 1
 }
+
+@test "support: default support message result" {
+  "skip"
+
+  exec_graphql 'alice' 'support-messages'
+  language="$(graphql_output '.data.me.supportChat')"
+  
+  # expect an empty array
+  [[ "$language" == "[]" ]] || exit 1
+}
+
+@test "support: ask 2 questions" {
+  "skip"
```

Comment:
same here, I didnt mention before because it was a draft

Reply:
it's really to develop locally